posydon/utils/common_functions.py

```diff
@@ -933,20 +933,22 @@ def inspiral_timescale_from_orbital_period(star1_mass, star2_mass,
     return T_merge
 
 
-def spin_stable_mass_transfer(star_mass_preMT, star_mass_postMT):
+def spin_stable_mass_transfer(spin_i, star_mass_preMT, star_mass_postMT):
```

Comment:
On a quick search, I didn't found where in this function is called in the code. But I guess, you'd need to update all calls of that function when you add a new argument.

Reply:
yes, this function is only called in my latest change above in mesa_step